lib/webpack-tools/src/types.ts

```diff
@@ -0,0 +1,37 @@
+import type { Options, StorybookConfig as BaseStorybookConfig } from '@storybook/core-common';
+
+export type { Options, Preset, BuilderResult, TypescriptOptions } from '@storybook/core-common';
+
+export interface CommonWebpackConfiguration {
```

Comment:
FIXME comment: this is the bare minimum type that our common code expects all versions of webpack to implement. in the future we can update this as needed, but we will also need to update the utility code.